src/hardware/serialport/softmodem.cpp

```diff
@@ -708,55 +731,58 @@ void CSerialModem::Timer2(void) {
 				} else {
 					plusinc=0;
 				}
-	// If not a special pause command, should go for bigger blocks to send 
+	// If not a special pause command, should go for bigger blocks to send
 			}
 			tmpbuf[txbuffersize] = txval;
 			txbuffersize++;
 		}
 	} // while loop
-	
+
 	if (clientsocket && sendbyte && txbuffersize) {
 		// down here it saves a lot of network traffic
-		if(!clientsocket->SendArray(tmpbuf,txbuffersize)) {
+		if (!clientsocket->SendArray(tmpbuf,txbuffersize)) {
 			SendRes(ResNOCARRIER);
 			EnterIdleState();
 		}
 	}
 	// Handle incoming to the serial port
-	if(!commandmode && clientsocket && rqueue->left()) {
+	if (!commandmode && clientsocket && rqueue->left()) {
 		usesize = rqueue->left();
-		if (usesize>16) usesize=16;
-		if(!clientsocket->ReceiveArray(tmpbuf, &usesize)) {
+		if (usesize > 16)
+			usesize = 16;
+		if (!clientsocket->ReceiveArray(tmpbuf, &usesize)) {
 			SendRes(ResNOCARRIER);
 			EnterIdleState();
-		} else if(usesize) {
-			// Filter telnet commands 
-			if(telnetmode) TelnetEmulation(tmpbuf, usesize);
-			else rqueue->adds(tmpbuf,usesize);
+		} else if (usesize) {
+			// Filter telnet commands
+			if (telnetmode)
+				TelnetEmulation(tmpbuf, usesize);
+			else
+				rqueue->adds(tmpbuf,usesize);
 			cmdpause = 0;
-		} 
+		}
 	}
 	// Check for incoming calls
 	if (!connected && !waitingclientsocket && serversocket) {
-		waitingclientsocket=serversocket->Accept();
-		if(waitingclientsocket) {	
-			if(!CSerial::getDTR()) {
+		waitingclientsocket.reset(serversocket->Accept());
+		if (waitingclientsocket) {
+			if( !CSerial::getDTR()) {
```

Comment:
Seems like space and bracket switched spaces :)

Reply:
Good eye! I found myself getting dizzy after wading through the code trying to catch all the strange no-space formatting. I can't wait to get that clang reformatting tool in place to mass-apply this type of cleanup robotically (and 100% reliably).

Reply:
Fixed in 8e9e4a41